commands/miscellaneous/getStudentProfile.js

```diff
@@ -6,61 +6,60 @@ const ALL_GUILD_IDS = [BATId, BALId, BAGId, BADId, guildId];
 
 module.exports = {
   data: new SlashCommandBuilder()
-    .setName('getstudentprofile')
-    .setDescription('Gets info of a students across BA servers')
-    .setDefaultPermission(false)
-    .addUserOption(option =>
-      option.setName('user')
-        .setDescription('The user you want to get their profile.')
-        .setRequired(true)),
+      .setName('getstudentprofile')
+      .setDescription('Gets info of a students across BA servers')
+      .setDefaultPermission(false)
+      .addUserOption(option =>
+          option.setName('user')
+              .setDescription('The user you want to get their profile.')
+              .setRequired(true)),
   async execute(interaction) {
     const options = interaction.options;
     const user = options.getUser('user');
     const fields = [];
 
-    const formatStudentInfo = (guildName, value) => {
+    const formatField = (displayName, value) => {
       const info = {
-        name: `Roles at ${guildName}`,
+        name: `${displayName}: `,
         value: `${value}`,
-        inline: true
+        inline: false
```

Comment:
i personally like this set to true for readability because some folks have a loooooot of tags 😆. not sure if it matters to those using the command regularly? you could default it to true and then send a false flag in with the displayName and createdAtDate so it stays on bottom of the embed. 

```
    const formatField = (displayName, value, inline = true) => {
      const info = {
        name: `${displayName}: `,
        value: `${value}`,
        inline: inline
      };
      return info;
    };
```

then 

```
fields.unshift(formatField('Display Name', `${displayName}`, false));
fields.push(formatField('Created', `${createdAtDate} UTC`, false));
```